app/view_models/workarea/storefront/search_autocomplete_product_view_model.rb

```diff
@@ -0,0 +1,13 @@
+module Workarea
+  module Storefront
+    class SearchAutocompleteProductViewModel < RecommendationsViewModel
```

Comment:
Since this doesn't wrap a single product (it presents a collection of them), I think it'd make more sense to reflect this in the class name. Like `SearchAutocompleteProductsViewModel` or `SearchAutocompleteRecommendationsViewModel` or just something plural.